install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -69,6 +87,20 @@ roleRef:
   name: istio-pilot
   apiGroup: rbac.authorization.k8s.io
 ---
+# Grant permissions to Mixer.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: istio-mixer-admin-role-binding
```

Comment:
There's already a binding of `istio-mixer-service-account` to `istio-pilot` role.
Maybe you don't have to edit this file?

Reply:
I removed that binding. every component gets own role / service account now.